Redesign multiple volumegroups (#146)
cybozu-go/topolvm: docs/proposals/multiple-volumegroups.md

provisioner: topolvm.io
parameters:
  "csi.storage.k8s.io/fstype": "xfs"
  "topolvm.io/device-class": "ssd"
volumeBindingMode: WaitForFirstConsumer
```

> `lvmd` receives the device class as a parameter.
> `lvmd` has a setting for each device class, can decide a target volume group for a given device class.
> `lvmd` will operate a logical volume on the target volume group.

Reviewer: ```suggestion
The device class name is then passed to `lvmd`.

`lvmd` has a mapping between device classes and LVM volume groups.
It can, therefore, create a logical volume in multiple volume groups.
```

> If a device class is empty, `lvmd` will use default volume group.

Reviewer: ```suggestion
If no device class is given, `lvmd` will use the default volume group.
```

    resources:
      requests:
        topolvm.io/capacity: "1"
      limits:

>         topolvm.io/capacity: "1"

Reviewer: I think this is extraneous.
